src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -34,15 +35,53 @@ public ViewCommand(String tele) {
         this.tele = tele.toLowerCase();
     }
 
+    /**
+     * Checks if char array given contains any illegal characters (not letters, numbers or underscore)
+     * @param arr char array
+     * @return boolean value
+     */
+    public boolean containsIllegalCharacters(char[] arr) {
```

Comment:
This method is likely redundant. It will probably be better to use the existing `isValidTelegram` public static method in the Telegram class, so that we have a sole point of change if we ever want to change the requirements